src/libraries/System.CodeDom/src/System.CodeDom.csproj

```diff
@@ -3,6 +3,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <DefineConstants>$(DefineConstants);CODEDOM</DefineConstants>
     <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;net461</TargetFrameworks>
+    <Nullable>annotations</Nullable>
```

Comment:
To declare ourselves complete with annotations, I guess we need to change 'annotations' to 'enable' at some point?

Reply:
Yes